### DISCUSSION TITLE
API Daftar Tanaman

### DISCUSSION THREAD

**Discussion by rynn04** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#328] [General]
Tolong tambahin API untuk daftar tanaman di indonesia. Terima kasih sebelumnya

**Comment by wawan8666** [comment]
tolong carikan api digipos

**Comment by LemonSync** [comment]
> Tolong tambahin API untuk daftar tanaman di indonesia. Terima kasih sebelumnya

1. https://perenual.com/docs/api
2. https://www.postman.com/api-evangelist/openthc/request/6lxv9e2/get-a-list-of-plants

Itu bisa gw rasa
